ament_cppcheck/ament_cppcheck/main.py

```diff
@@ -86,6 +91,9 @@ def main(argv=sys.argv[1:]):
            '--xml-version=2']
     if args.language:
         cmd.extend(['--language={0}'.format(args.language)])
+    if args.include_dirs:
+        for include_dir in args.include_dirs:
+            cmd.extend(['-I', '{0}'.format(include_dir)])
```

Comment:
Why use a format string here when `include_dir` already contains the desired string?

Reply:
Left over from a previous iteration. I'll change it.